internal/settings/settings.go

```diff
@@ -2,13 +2,15 @@ package settings
 
 import (
 	"fmt"
+
 	"github.com/mitchellh/mapstructure"
 )
 
 type Options struct {
 	// RootModulePaths describes a list of absolute paths to root modules
 	RootModulePaths    []string `mapstructure:"rootModulePaths"`
 	ExcludeModulePaths []string `mapstructure:"excludeModulePaths"`
+	ID                 string   `mapstructure:"id"`
```

Comment:
Because this is exposed to the client and essentially becoming part of a public API, could we give it a slightly more descriptive name - e.g. `CommandPrefix` (`mapstructure:commandPrefix`)? If we ever need to use a unique ID for something else, then we can just add that as a separate option, but I would hope this is the only one we need to support for the foreseeable future. 😄

Reply:
I sort of feel strongly to the opposite approach. I wanted to be very careful about the things we bake into essentially our own protocol we have to live with. I thought the concept of assigning the server an `ID` would be very flexible and canonical, I wanted to keep the details of command suffixing out of it if possible.

But I do agree 🤔 having something called `ID` then be implicitly suffixed is sorta of a weird piece of knowledge client developers need to know about is also weird... I could maybe be convinced of the otherside, I need to sleep on it/discuss further with you/team.